efcsydney/efcsydney-roster
# #441 Ability to audit the change logs on Admin

![image](https://user-images.githubusercontent.com/136648/45253656-23bbed00-b3ae-11e8-9f94-c8c365a44722.png)

As an admin
I would like to audit the data updated events
so that I can tell if it's a reasonable change by someone

#### Change Logs

* API - Introducing GraphQL
* Frontend - Using Apollo.js to query the GraphQL
* Trigger the Pusher events for future use

#### QA URL

* https://demo-roster.efcsydney.org/graphql - Query Interface
* https://demo-roster.efcsydney.org/#/admin/changelogs - Admin Interface

#### QA Steps

1. Update any data such as event, service, or serviceInfo.
1. Go to the Change Logs Admin page
1. Ensure it reflects the change you just made
